core/src/main/scala/io/chrisdavenport/npmpackage/sbtplugin/NpmPackagePlugin.scala

```diff
@@ -168,6 +168,9 @@ object NpmPackagePlugin extends AutoPlugin {
     val npmPackageBinaries: SettingKey[Seq[(String, String)]] =
       settingKey("The name of the binary executable - defaults to project name")
 
+    val npmPackageType: SettingKey[String] = 
+      settingKey("The type of the package - defaults to 'commonjs' for ModuleKind.CommonJSModule and 'module' for ModuleKind.ESModule")
```

Comment:
Maybe also clarify `CommonJS` is the default for no modules as well.

Reply:
Yes good point. Added